replication-api/src/main/java/org/codice/ditto/replication/api/NotFoundException.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ditto.replication.api;
+
+/** Persistence exception indicating when a data entry that should be stored couldn't be found. */
+public class NotFoundException extends RuntimeException {
```

Comment:
Added this because we were previously using the java web service not found exception and it would give a 404 exception message which was misleading.